CourierMQTT/MQTT/Connection/MQTT-Client-Framework/MQTTClientFrameworkConnection.swift

```diff
@@ -72,11 +72,24 @@ class MQTTClientFrameworkConnection: NSObject, IMQTTConnection {
         self.connectOptions = connectOptions
 
         let port = Int(connectOptions.port)
+        let isTls = false
```

Comment:
to simplify, we can just replace this if conditions from:

```swift
if port == 443 {
    securityPolicy = MQTTSSLSecurityPolicy()
    securityPolicy?.allowInvalidCertificates = true
}
```

to

```swift
 if scheme == "ssl" {
     securityPolicy = MQTTSSLSecurityPolicy()
     securityPolicy?.allowInvalidCertificates = true
 }
```

so no need to declare `let isTls` and  `switch connectOptions.scheme`

Reply:
Done mas. In addition, I add the `useSSLWithTLS` to check the condition. Thank you